tests/ui_tools/test_popups.py

```diff
@@ -520,14 +520,14 @@ def test_init(self, msg_box: MessageBox) -> None:
         assert self.full_rendered_message.footer.widget_list == msg_box.footer
 
```

Comment:
Ah, is that so? I can't seem to recall that discussion.
The change to the commit text sounds better anyways.
Glad to know that the rest is fine.

Reply:
I've not rechecked, but was likely referring to Sashank's comment [here](https://github.com/zulip/zulip-terminal/pull/1537#issuecomment-2248558981), where he is expecting this will be necessary for spoilers.

Reply:
Ah, that's covered by this PR. Won't need to pass around any data for that.